### PR TITLE
docs: fix invalid link and unify link presentation

### PR DESCRIPTION
**Pull request by devzom**
There was an invalid link on 'automation token' which leads to 404, also not unified style of link presentation because other links have quotes. 

Use the mdx anchor tag [create-token] to link to proper document.

Prettify mdx file.

### CODE REVIEW

File: content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx [hunk 1]
```diff
@@ -47,13 +47,13 @@ Example:
...
 
-For more information, see "[Creating and viewing authentication tokens](creating-and-viewing-access-tokens)".
+For more information, see "[Creating and viewing authentication tokens][create-token]".
 
 ## Set the token as an environment variable on the CI/CD server
 
 Set your token as an environment variable, or a secret, in your CI/CD server.
 
-For example, in GitHub Actions, you would [add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).  Then you can make the secret available to workflows.
+For example, in GitHub Actions, you would [add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets). Then you can make the secret available to workflows.
```
Comment (wraithgar) [inline content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx:56]:
```suggestion
For example, in GitHub Actions, you would "[add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)". Then you can make the secret available to workflows.
```

Comment (devzom) [reply content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx]: That's right, didn't noticed this one.

File: content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx [hunk 2]
```diff
@@ -22,7 +22,7 @@ For more information on creating access tokens, including CIDR-whitelisted token
 
 ### Continuous deployment
 
-Since continuous deployment environments usually involve the creation of a deploy artifact, you may wish to create an [automation token](create-token#creating-tokens-on-the-website) on the website.  This will allow you to publish even if you have two-factor authentication enabled on your account.
+Since continuous deployment environments usually involve the creation of a deploy artifact, you may wish to create an "[automation token][create-token]" on the website. This will allow you to publish even if you have two-factor authentication enabled on your account.
```
[the remark this answers is not in the document]
Comment (devzom) [reply content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx:25]: I see the point, thank You. Let's rollback the quotes here

[the remark this answers is not in the document]
Comment (devzom) [reply content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx]: Fixed it, please review again

File: content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx [hunk 3]
```diff
@@ -47,13 +47,13 @@ Example:
...
 
-For more information, see "[Creating and viewing authentication tokens](creating-and-viewing-access-tokens)".
+For more information, see "[Creating and viewing authentication tokens][create-token]".
 
 ## Set the token as an environment variable on the CI/CD server
 
 Set your token as an environment variable, or a secret, in your CI/CD server.
 
-For example, in GitHub Actions, you would [add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).  Then you can make the secret available to workflows.
+For example, in GitHub Actions, you would "[add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)". Then you can make the secret available to workflows.
```
Comment (ljharb) [inline content/integrations/integrating-npm-with-external-services/using-private-packages-in-a-ci-cd-workflow.mdx:56]:
```suggestion
For example, in GitHub Actions, you would [add your token as a secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets). Then you can make the secret available to workflows.
```
quotes don't make sense here

**Review by devzom — COMMENTED** [review]
Done

**Pull request merged**